tgui/packages/tgui/interfaces/CharacterPreferences/index.tsx

```diff
@@ -0,0 +1,178 @@
+          <Tabs.Tab onClick={() => act('traits-window')}>Traits</Tabs.Tab>
+          <Tabs.Tab
+            selected={menu === CharacterPreferencesTabKeys.GameSettings}
+            onClick={() => setMenu(CharacterPreferencesTabKeys.GameSettings)}>
+            Game Settings
+          </Tabs.Tab>
+          <Tabs.Tab
+            selected={menu === CharacterPreferencesTabKeys.Saves}
+            onClick={() => setMenu(CharacterPreferencesTabKeys.Saves)}>
+            Cloud Saves
+          </Tabs.Tab>
+        </Tabs>
+        {menu === CharacterPreferencesTabKeys.General || menu === CharacterPreferencesTabKeys.Character ? (
+          <Flex>
+            <Flex.Item grow="1">
+              <Box>
+                {menu === CharacterPreferencesTabKeys.General && <GeneralTab />}
+                {menu === CharacterPreferencesTabKeys.Character && <CharacterTab />}
+              </Box>
+            </Flex.Item>
+            <Flex.Item ml="5px">
+              <Section fill>
+                <ByondUi
+                  params={{
+                    id: data.preview,
+                    type: 'map',
+                  }}
+                  style={{
+                    width: '80px',
+                    height: '80px',
+                  }}
+                />
+                <Box textAlign="center" mt="5px">
+                  <Button icon="chevron-left" onClick={() => act('rotate-clockwise')} />
+                  <Button icon="chevron-right" onClick={() => act('rotate-counter-clockwise')} />
+                </Box>
+              </Section>
+            </Flex.Item>
+          </Flex>
+        ) : (
+          <Fragment>
+            {menu === CharacterPreferencesTabKeys.GameSettings && <GameSettingsTab />}
+            {menu === CharacterPreferencesTabKeys.Saves && <SavesTab />}
+          </Fragment>
+        )}
+
+        <Section mt="5px">
+          <Button.Confirm content="Reset All" onClick={() => act('reset')} />
+        </Section>
+      </Window.Content>
+    </Window>
+  );
+};
+
+const SavesAndProfile = (_props, context) => {
+  const { act, data } = useBackend<CharacterPreferencesData>(context);
+
+  const activeProfileIndex = data.profiles.findIndex((p) => p.active);
+
+  return (
+    <Fragment>
+      <Flex mb="5px">
+        {data.profiles.map((profile, index) => (
+          <Flex.Item key={index} ml={index === 0 ? '0' : '5px'} grow="1">
+            <Profile profile={profile} index={index} />
+          </Flex.Item>
+        ))}
+      </Flex>
+
+      <Section>
+        <LabeledList>
+          <LabeledList.Item
+            label="Profile Name"
+            buttons={
+              activeProfileIndex > -1 ? (
+                <Fragment>
+                  <Button onClick={() => act('load', { index: activeProfileIndex + 1 })}>Reload</Button> -{' '}
+                  <Button onClick={() => act('save', { index: activeProfileIndex + 1 })}>Save</Button>
```

Comment:
Not sure you need a ` - ` between buttons here. If you do, probably write as:
```suggestion
                  <Button onClick={() => act('load', { index: activeProfileIndex + 1 })}>Reload</Button>
                  {' - '}
                  <Button onClick={() => act('save', { index: activeProfileIndex + 1 })}>Save</Button>
```